boards.txt

```diff
@@ -57,7 +57,7 @@ feather52832.menu.debug.l1.build.debug_flags=-DCFG_DEBUG=1
 feather52832.menu.debug.l2=Level 2 (Full Debug)
 feather52832.menu.debug.l2.build.debug_flags=-DCFG_DEBUG=2
 feather52832.menu.debug.l3=Level 3 (Segger SystemView)
-feather52832.menu.debug.l3.build.debug_flags=-DCFG_DEBUG=3
+feather52832.menu.debug.l3.build.debug_flags=-DCFG_DEBUG=3 -DCFG_SYSVIEW=0xFFFFFFFF
```

Comment:
Please remove all of `-DCFG_SYSVIEW=0xFFFFFFFF` , CFG_DEBUG level will by default enable it. This can done within an header/config while with a single ifdef rather than adding it to all of the board variant.

```
#if CFG_DEBUG >= 3
#define CFG_SYSVIEW   0xFFFFFFFF
#endif 
```

Reply:
done.